### PR TITLE
docs(index):make more friendly to RPC user

### PR DESCRIPTION
**Pull request by scvoet**
In addition to the change information on the title the, and I also made more details.

## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

### CODE REVIEW

File: source/zh-cn/docs/index.md [hunk 1]
```diff
@@ -36,34 +37,35 @@ Hexo 是一个快速、简洁且高效的博客框架。Hexo 使用 [Markdown](h
...
 
-{% note info Windows 用户 %}
-对于中国大陆地区用户，可以前往 [淘宝 Git for Windows 镜像](https://npm.taobao.org/mirrors/git-for-windows/) 下载 git 安装包。
-{% endnote %}
-
 ### 安装 Node.js
 
-Node.js 为大多数平台提供了官方的 [安装程序](https://nodejs.org/en/download/)。对于中国大陆地区用户，可以前往 [淘宝 Node.js 镜像](https://npm.taobao.org/mirrors/node) 下载。
+Node.js 为大多数平台提供了官方的 [安装程序](https://nodejs.org/en/download/)。
+> 中国大陆地区用户：[淘宝 Node.js 镜像](https://npm.taobao.org/mirrors/node)
```
Comment (SukkaW) [inline source/zh-cn/docs/index.md]:
```suggestion
{% note info 中国大陆地区用户 %}
中国大陆地区的用户可以前往淘宝提供的 [Node.js 镜像](https://npm.taobao.org/mirrors/node) 下载。
{% endnote %}
```

File: source/zh-cn/docs/index.md [hunk 2]
```diff
@@ -36,34 +37,35 @@ Hexo 是一个快速、简洁且高效的博客框架。Hexo 使用 [Markdown](h
...
 如果在尝试安装 Hexo 的过程中出现 `EACCES` 权限错误，请遵循 [由 npmjs 发布的指导](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally) 修复该问题。强烈建议 **不要** 使用 root、sudo 等方法覆盖权限
 {% endnote %}
 
 ### 安装 Hexo
 
 所有必备的应用程序安装完成后，即可使用 npm 安装 Hexo。
+> 中国大陆地区用户建议使用 cnpm 替代 npm，则以后涉及 npm 命令应均由 cnpm 替代。
+> ```
+> npm install -g cnpm --registry=https://registry.npm.taobao.org
+> ```
```
Comment (SukkaW) [inline source/zh-cn/docs/index.md]: Personally I won't suggest `cnpm` in Hexo's documents. `cnpm` cli is using different package resolve strategy which sometime breaks `node_modules`.

```bash
alias cnpm="npm --registry=https://registry.npm.taobao.org \
--cache=$HOME/.npm/.cache/cnpm \
--disturl=https://npm.taobao.org/dist \
--userconfig=$HOME/.cnpmrc"
```

Please use this instead.

File: source/zh-cn/docs/index.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ Hexo 是一个快速、简洁且高效的博客框架。Hexo 使用 [Markdown](h
 ### 安装 Git
 
 - Windows：下载并安装 [git](https://git-scm.com/download/win).
+  - 中国大陆地区用户：[淘宝 Git for Windows 镜像](https://npm.taobao.org/mirrors/git-for-windows/)
```
Comment (SukkaW) [inline source/zh-cn/docs/index.md]: IMHO, a separate `blockquote` is preferred.

Comment (scvoet) [reply source/zh-cn/docs/index.md]: > IMHO, a separate is preferred.`blockquote`

@SukkaW  Emm, Taobao only provides the git mirror of Windows. May it isn't suitable to use a blockquote. Use the list should be more appropriate. Do you have another suggestion for this?

Comment (SukkaW) [reply source/zh-cn/docs/index.md]: So why not just retain current:

```
{% note info Windows 用户 %}	
对于中国大陆地区用户，可以前往 [淘宝 Git for Windows 镜像](https://npm.taobao.org/mirrors/git-for-windows/) 下载 git 安装包。
{% endnote %}
```

Comment (scvoet) [reply source/zh-cn/docs/index.md]: @SukkaW I had changed it.

File: source/zh-cn/docs/index.md (hunk 2 above)
Comment (scvoet) [reply source/zh-cn/docs/index.md]: @SukkaW I hadn't use the way. Do you want to change the part of the docs?

File: source/zh-cn/docs/index.md [hunk 4]
```diff
@@ -27,7 +27,12 @@ Hexo 是一个快速、简洁且高效的博客框架。Hexo 使用 [Markdown](h
 
 ### 安装 Git
 
-- Windows：下载并安装 [git](https://git-scm.com/download/win).
+- Windows：下载并安装 [git](https://git-scm.com/download/win)。
+
+{% note info Windows 用户 %}
```
Comment (SukkaW) [inline source/zh-cn/docs/index.md]:
```suggestion
{% note info 中国大陆地区用户 %}	
```

File: source/zh-cn/docs/index.md [hunk 5]
```diff
@@ -85,4 +93,3 @@ $ npm install hexo
   ``` bash
   echo 'PATH="$PATH:./node_modules/.bin"' >> ~/.profile
   ```
```
Comment (SukkaW) [inline source/zh-cn/docs/index.md]:
```suggestion
  ```

```

File: source/zh-cn/docs/index.md [hunk 6]
```diff
@@ -36,34 +41,37 @@ Hexo 是一个快速、简洁且高效的博客框架。Hexo 使用 [Markdown](h
...
 如果在尝试安装 Hexo 的过程中出现 `EACCES` 权限错误，请遵循 [由 npmjs 发布的指导](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally) 修复该问题。强烈建议 **不要** 使用 root、sudo 等方法覆盖权限
 {% endnote %}
 
 ### 安装 Hexo
 
 所有必备的应用程序安装完成后，即可使用 npm 安装 Hexo。
+> 中国大陆地区用户建议使用 cnpm 替代 npm，则以后涉及 npm 命令应均由 cnpm 替代。
+> ```
+> npm install -g cnpm --registry=https://registry.npm.taobao.org
+> ```
```
Comment (SukkaW) [inline source/zh-cn/docs/index.md]:
```suggestion
{% note info 中国大陆地区用户 %}
中国大陆地区用户可以考虑使用 [淘宝 NPM 镜像](https://npm.taobao.org/) 以加快安装速度。
{% endnote %}
```

**Pull request closed without merging**